lib/accelerator.js

```diff
@@ -11,15 +11,16 @@ const didv1 = require('did-veres-one');
 const fs = require('fs');
 const forge = require('node-forge');
 const logger = require('./logger');
+const {promisify} = require('util');
 
 require('bedrock-express');
 
 // module API
 const api = {};
 module.exports = api;
 
-bedrock.events.on('bedrock-express.ready', (app, callback) =>
-  _setupAccelerator(callback));
+bedrock.events.on(
+  'bedrock-express.ready', () => promisify(_setupAccelerator)());
```

Comment:
Fine with this here for this release... I thought we were going to put the accelerator on its own site?

Reply:
I've added an item to the roadmap for removing this old accellerator code.